packages/design-system/src/components/toast.tsx

```diff
@@ -0,0 +1,115 @@
+import * as ToastPrimitive from "@radix-ui/react-toast";
+import BaseToast, { useToaster } from "react-hot-toast/headless";
+import { keyframes, styled } from "../stitches.config";
+import { CheckIcon, Cross1Icon } from "@webstudio-is/icons";
+import { Box } from "./box";
+
+const VIEWPORT_PADDING = "$2";
+
+const hide = keyframes({
+  "0%": { opacity: 1 },
+  "100%": { opacity: 0 },
+});
+
+const slideIn = keyframes({
+  from: { transform: `translateY(calc(0px - ${VIEWPORT_PADDING}px))` },
+  to: { transform: "translateY(0)" },
+});
+
+const swipeOut = keyframes({
+  from: { transform: "translateX(var(--radix-toast-swipe-end-x))" },
+  to: { transform: `translateX(calc(100% + ${VIEWPORT_PADDING}px))` },
+});
+
+const StyledViewport = styled(ToastPrimitive.Viewport, {
+  position: "absolute",
+  top: 0,
+  right: 0,
+  display: "flex",
+  flexDirection: "column",
+  padding: VIEWPORT_PADDING,
+  gap: VIEWPORT_PADDING,
+  minWidth: 200,
+  width: "auto",
+  maxWidth: "100vw",
+  margin: 0,
+  listStyle: "none",
+  zIndex: "$max",
+  outline: "none",
+});
+
+const StyledToast = styled(ToastPrimitive.Root, {
+  borderRadius: 6,
+  boxShadow:
+    "hsl(206 22% 7% / 35%) 0 $space$2 $space$6 -$space$2, hsl(206 22% 7% / 20%) 0 $space$2 $space$4 -$space$3",
```

Comment:
Is there a way to add our colors with transparency?

Reply:
hmm not sure how to deal with it, we can  define colors in hsl and then have a function that changes opacity parameter, but we need to discuss with @taylornowotny when would you use that and why

Reply:
I was thinking RGBA, but that's an implementation detail.

For shadows and stuff we should definitely have something that allows to pass transparency

Reply:
the question is when do you need to pass a non-static, predefined transparency

Reply:
I do agree, at remote we had these predefined so for example $highContrast_20 would be 20% opacity